sda/cmd/api/api_test.go

```diff
@@ -710,6 +710,37 @@ func testEndpoint(c *gin.Context) {
 	c.JSON(200, gin.H{"ok": true})
 }
 
+func (s *TestSuite) TestLogLevelDebug() {
+	// Capture log output from gin in buf
+	var buf bytes.Buffer
+	log.SetOutput(&buf)
+	// Ensure we log to stdout after this test is completed
+	defer log.SetOutput(os.Stdout)
+
+	log.SetLevel(log.DebugLevel)
+	srv := setup(Conf)
+	srv.Addr = ":8080" // TODO: What is the proper way to configure this from api_test.go?
```

Comment:
On normal startup the `config.NewConfig("api")` is called which sets the default host:port to `0.0.0.0:8080`
In the test suite we could set the defaults in the `SetupSuite` function.
```
Conf.API.Host = "0.0.0.0"
Conf.API.Port = 8080
```